### ISSUE TITLE
Working on xbim for mobile

### ISSUE THREAD

**Issue by gijskoning** [xBimTeam/XbimWebUI#108]
Hi i'm trying to get xbimWebUI working on mobile.
When looking through the code i find that navigate is a local function in an local function of xviewer. Why not change this and make it an local function of xviewer. Than it's much easier to implement for example touch functions. 
Im new with Xbim so i'll just post it as an issue rather than an pull request.

**Comment by bimbangboom** [comment]
https://github.com/xBimTeam/XbimWebUI/issues/84

You can see an existing issue addresses this same modification.  I think it's a low priority for the xBIM team, so I will probably start looking at it soon.

**Comment by martin1cerny** [comment]
You are absolutely right @bimbangboom . Any kind of contribution is welcome and appreciated. It is one of the things I'd like to do but can't get to do it for a while. I'll happily merge in the suggested change to make `navigate()` public function of `xViewer`.

**Comment by zelima93** [comment]
is this alive?
im trying xBim for mobile too

**Comment by martin1cerny** [comment]
Touch navigation is implemented in `develop` branch thanks to the contribution of @GeorgDangl.

**Comment by martin1cerny** [comment]
Also, `navigate()` function is now public in `develop` branch so I'm closing this issue.

**Comment by save2004** [comment]
forgive me if I write in a closed post.
I have only recently started using Xbim, but I need to guarantee mobile functionality and the @GeorgDangl solution seems to me to resolve the issue.
How can I integrate it into my project? Is there a guide?
I saw at this link:
https://github.com/xBimTeam/XbimWebUI/commit/9a8c2c8b057d87b2cbd2a009523d54f2b5cc7df9
the files affected by the changes, but I can't find them in the Script folder of my project ...
Thank you for your help and for the work you do!
